support-frontend/assets/components/page/heroRoundel.jsx

```diff
@@ -20,13 +20,17 @@ const heroRoundelStyles = css`
   /* Do not remove float! It makes the circle work! See link below */
   float: left;
   transform: translateY(-67%);
-  min-width: ${roundelSizeMob}px;
-  max-width: ${roundelSize}px;
-  width: calc(100% + ${space[3]}px);
+  min-width: 100px;
```

Comment:
Is it worth making this also `roundelSizeMob`, to avoid the magic number?

Reply:
Yes, good point